### PR TITLE
chore: auto expire nonce cache

### LINKED ISSUE #20: Redis cache invalidation for nonces

### PR DESCRIPTION
**Pull request by jleach**
Automatically expire the nonce after a period of time.

Fixes #20

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 app_id = 'L796QSLV3E.ca.bc.gov.BCWallet'
 rp_id_hash_end = 32
 counter_start = 33
 counter_end = 37
 aaguid_start = 37
 aaguid_end = 53
 cred_id_start = 55
+
+## Redis
+auto_expire_nonce = 60 * 60 * 10 # 10 minutes
```
Comment (bryce-mcmath) [inline src/constants.py:12]: Is this milliseconds or seconds? Either way I don't think that's ten minutes. Looks like it's either 36 seconds or ten hours

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Just a tweak of the expire time

File: src/constants.py (hunk 1 above)
Comment (jleach) [reply src/constants.py:12]: 10 hours. it's seconds. Will fix.

Comment (jleach) [reply src/constants.py]: Fixed.

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**